include/CL/sycl/device.hpp

```diff
@@ -248,13 +248,13 @@ class device
 
 template <>
 inline auto device::get_info<info::device::max_work_group_size>() const {
-  return size_t { 63 };
+  return size_t { 8 };
 }
 
 
 template <>
 inline auto device::get_info<info::device::max_compute_units>() const {
-  return size_t { 56 };
+  return size_t { 8 };
```

Comment:
And now with these 2 previous changes this unit test is failing:
```
/home/keryell/Xilinx/Projects/OpenCL/SYCL/triSYCL/branch/tmp/tests/device/get_info.cpp(14): test d.get_info<info::device::max_work_group_size>() == 63 failed in function: 'int test_main(int, char**)'
``` 
and so on.
Please update the test too.